app/src/main/java/com/willowtree/vocable/presets/PhrasesFragment.kt

```diff
@@ -51,8 +51,8 @@ class PhrasesFragment : BaseFragment() {
             val phraseButton =
                 PhraseButtonBinding.inflate(inflater, binding?.phrasesContainer, false)
             with(phraseButton.root as VocableButton) {
-                // TODO: Use currently set Locale
-                text = phrase.localizedUtterance[Locale.US.language]
+                val pair = phrase.getLocalizedPair()
```

Comment:
You can destructure this pair to make it more readable. `val (locale, phrase) = phrase.getLocalizedPair()` so that in the set text you dont have to do `setText(pair.first, pair.second)` you just do `setText(locale, phrase)`.

Reply:
Oh nice! I didn't know this was a thing in Kotlin